share/wake/lib/system/job.wake

```diff
@@ -381,16 +381,25 @@ export def mkJobCacheRunner ((Runner name score baseDoIt): Runner): Runner =
 
             def jobCacheVisible = JArray (map mkVisJson vis)
 
+            # TODO: Allow this to be configurable
+            def workspace = "/workspace"
+
+            def abs_dir =
+                if matches `/.*` dir then
+                    dir
+                else
+                    "{workspace}/{dir}"
+
             def jobCacheJsonIn =
                 prettyJSON
                 $ JObject (
-                    "cwd" :-> JString dir,
+                    "cwd" :-> JString abs_dir,
                     "command_line" :-> JString cmd.implode,
                     "envrionment" :-> JString env.implode,
                     "stdin" :-> JString stdin,
                     "input_files" :-> jobCacheVisible,
                     "client_cwd" :-> JString client_cwd,
-                    "dir_redirects" :-> JObject ("/workspace" :-> JString "./",),
+                    "dir_redirects" :-> JObject (workspace :-> JString "./",),
```

Comment:
`workspace` is already an absolute path

Reply:
Ok I figured out what you were talking about here because of a bug I had to fix. I had shadowed the magic workspace variable to be something else not this. After I got rid of that my code still compiled! That path was previously a sandbox path not as I think you understood it a client path.